docs/source/SARS-CoV-2_resources.rst

```diff
@@ -0,0 +1,11 @@
+===================================
```

Comment:
Not sure if this file is accessible on the RTDs page as it doesn't seem to be referenced by any other page